_content/tour/eng/methods.article

```diff
@@ -147,7 +147,7 @@ use of pointer semantics because the data dictates the data semantics and not th
 function.
 
 One exception to using value semantics is when you need to share a slice or map
-with a function that performs unmarshaling or decoding.
```

Comment:
@hrvadl `unmarshaling` (with one `l`) is completely correct in Go world. From https://go.dev/wiki/Spelling:

> - use marshaling, marshaled etc (single ell). See https://go.dev/cl/33017

Reply:
I know, but it's ok that both spellings work.